core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -313,7 +313,8 @@ object SparkEnv extends Logging {
     // Let the user specify short names for shuffle managers
     val shortShuffleMgrNames = Map(
       "hash" -> "org.apache.spark.shuffle.hash.HashShuffleManager",
-      "sort" -> "org.apache.spark.shuffle.sort.SortShuffleManager")
+      "sort" -> "org.apache.spark.shuffle.sort.SortShuffleManager",
+      "unsafe" -> "org.apache.spark.shuffle.unsafe.UnsafeShuffleManager")
```

Comment:
i talked with patrick, and we thought tungsten-sort could be a good name for this